PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,73 @@
+<!--
+  If you have a relevant JIRA issue number, please put it in the issue title.
+  Example: MODORDERS-70 Orders schema updates
```

Comment:
Please specify as an example the ui-inventory project. Same [here](https://github.com/folio-org/ui-inventory/pull/969/files).